### PR TITLE
Implement convenience method for forking a new browser instance

### PR DESCRIPTION
**Pull request by scriby**
Addresses #27

### CODE REVIEW

**Review by shermaneric — APPROVED** [review]
Changes look great and new test passes when running. Approving!

File: app/browser-sync.ts [hunk 1]
```diff
@@ -83,6 +83,29 @@ export class BrowserSync {
...
   }
+
+  forkNewDriverInstance(useSameUrl?: boolean, copyMockModules?: boolean, copyConfigUpdates?: boolean) {
+    const newBrowser = this.getBrowser().forkNewDriverInstance(useSameUrl, copyMockModules, copyConfigUpdates);
+
+    const waitForAngularEnabled = this.waitForAngularEnabled();
+
+    //Use the runner class to reset global variables after spawning the new instance
+    const runner = new Runner(this.getProcessedConfig());
+    runner.setupGlobals_(newBrowser);
```
Comment (mindywhitsitt) [inline app/browser-sync.ts:94]: So unlike Angular's forkNewDriverInstance, which just returns the new instance, this is more like forkAndSwitchToNewDriverInstance? Should we make that more explicit somehow?

**Review by mindywhitsitt — APPROVED** [review]

File: app/browser-sync.ts (hunk 1 above)
Comment (scriby) [reply app/browser-sync.ts:94]: i couldn't think of a use case where you want to open a new instance and not use it, so I felt like this is a better default.

The consistency argument is still good though. I am ok either way.

Comment (mindywhitsitt) [reply app/browser-sync.ts:94]: I agree this is a reasonable default. And it's exactly our use-case.

I do think the consistency with Protractor is an issue though. I think it's especially confusing to have a method with the same name but different behavior, so we'll go ahead and rename this method.

**Comment by mindywhitsitt** [comment]
Oh, and thank you so much for adding this, @scriby . :)

**Pull request merged**